dev/com.ibm.ws.org.apache.aries.jndi.core/bnd.overrides

```diff
@@ -14,6 +14,8 @@ bVersion=1.1
 
 Bundle-SymbolicName: com.ibm.ws.org.apache.aries.jndi.core
 
+Import-Package: com.sun.naming.internal;resolution:="optional",*
```

Comment:
Is this package ever exported by the system bundle?

I think you are getting this from the boot delegation setting in `com.ibm.ws.kernel.launch.internal.FrameworkConfigurator.extraBootDelegationPackages(BootstrapConfig)` for "com.sun.*.

Reply:
Removed import